test/scripts/install-ovs.sh

```diff
@@ -58,6 +58,18 @@ fi
 
 echo -e "\033[35m Succeed to create vlan interface: ${HOST_ADDITIONAL_INTERFACE}.${VLAN30}、 ${HOST_ADDITIONAL_INTERFACE}.${VLAN40} in kind-node ${VLAN_GATEWAY_CONTAINER} \033[0m"
 
+
+# https://github.com/antrea-io/antrea/issues/51
+# fix: it possibley fails to insmod openvswitch.ko in the container in some OS version
+# so it could load the ko in the host os in advance to make sure the ovs service could be started in the container
+echo "=========install openvswitch in host os"
+sudo apt-get update 
+sudo apt-get install -y openvswitch-switch
+sudo modinfo openvswitch
```

Comment:
modprobe openvswith?

Reply:
systemctl start openvswitch-switch    will implement it